.github/workflows/snapshot-publish.yml

```diff
@@ -0,0 +1,67 @@
+name: Publish in-development builds from main
+
+on:
+  schedule:
+    # Run daily on week days
+    - cron:  '0 11,23 * * 1-5'
```

Comment:
Will this run even when there are no _new_ commits?

Reply:
Yes - but given that we usually have multiple commits per day, I suspect that's okay.

Reply:
(The hours (11+23 UTC) are outside of the "high commit traffic time")